bootstrap-salt.sh

```diff
@@ -3671,8 +3759,12 @@ install_centos_git_deps() {
         __PACKAGES="${__PACKAGES} python-libcloud"
     fi
 
-    # shellcheck disable=SC2086
-    __yum_install_noinput ${__PACKAGES} || return 1
+    if [ "${_INSTALL_PY}" = "${BS_TRUE}" ]; then
+        __install_python_and_deps || return 1
+    else
+        # shellcheck disable=SC2086
+        __yum_install_noinput ${__PACKAGES} || return 1
```

Comment:
There is not a repo that provides the appropriate python packages for the dependencies. For example the repository added does not provide python27-jinja2,python2-msgpack, etc. I only have access to the python26 packages, So I have had to install all the dependencies using pip that you can see in the `__install_python_and_deps` function. I did notice in those packages that I am missing the check to install libcloud if `-L` is passed. So I have added that in an extra commit for that. (although I haven't been able to test it because the epel repo is down right now)